packages/gatsby-theme-docs/src/layouts/internals/layout-page-with-hero.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { ThemeProvider } from 'emotion-theming';
+import { designSystem, Markdown } from '@commercetools-docs/ui-kit';
+
+// TODO: for now we assume that the website provides a file `homepage-hero.svg`
+// in the `static` folder.
+const homepageHeroSvgUri = '/homepage-hero.svg';
+
+// The container is used to render a background image with for the full page width.
+const Container = styled.div`
+  grid-area: page-header;
+  position: relative;
+  background-color: ${(props) => props.theme.colors.light.primary};
```

Comment:
I assume that as a fallback the background color should be the website primary color